package.json

```diff
@@ -17,18 +17,14 @@
     "responsive",
     "mobile-friendly",
     "front-end",
-    "less",
+    "sass",
     "modern"
   ],
   "bugs": {
     "url": "https://github.com/picturepan2/spectre/issues"
   },
   "devDependencies": {
     "gulp": "^3.9.1",
-    "gulp-clean-css": "^2.0.6",
-    "gulp-csscomb": "^3.0.7",
-    "gulp-less": "^3.0.5",
-    "gulp-rename": "^1.2.2",
-    "less-plugin-autoprefix": "^1.5.1"
+    "gulp-sass": "^3.0.5"
```

Comment:
@DJTB switched this out to gulp-sass 2.3.2 and it working again for me. Let me know if it's still an issue.